validphys2/src/validphys/pseudodata.py

```diff
@@ -267,6 +268,9 @@ def make_replica(
         # positivity control
         if np.all(shifted_pseudodata[full_mask] >= 0):
             return shifted_pseudodata
+        elif "POL" in fitbasis and trials == (max_tries // 100):
```

Comment:
Why do you need the second condition and you are not simply retuning?

Reply:
The 2nd return is only valid in the polarized case in which observable can be negative. In the unpolarized (n)PDF case, given that the observable must always be positive, if is not caught by the first `if` it should fail.

Reply:
But ofc, we also need to make sure that the negative obs are not due to random fluctuations, hence still the need for `trials == (max_tries // 100)`.